docs/proposals/advanced-nginx-extensions.md

```diff
@@ -445,6 +445,21 @@ one `http` snippet with multiple `http` snippets scattered around in the spec.
 
 NGF will not validate the values of snippets. See the next section.
 
+When a SnippetsFilter is specified in a routing rule and one of the following is true:
```

Comment:
Adding a link to the Gateway API spec documentation which gives more detail on this specifications would be helpful to me. But I can see how it may be unnecessary.

Reply:
Added a link